### PR TITLE
SFR-885 Add GA tracking events for Read Online and Download buttons

### PR DESCRIPTION
**Pull request by mwbenowitz**
[Jira Ticket](http://jira.nypl.org/browse/SFR-885)

### This PR does the following:
- Adds `onClick` events to the "Read Online" and "Download" buttons in edition cards that trigger GA tracking events that record which button was clicked, the source of the edition being accessed and the title of the work

### Testing requirements & instructions: 
-  Verify that the onClick triggers are present in all cases
- (Optional) trigger an event and verify that it appears in the Google Analytics reports

### CODE REVIEW

File: src/app/components/Card/EditionCard.jsx [hunk 1]
```diff
@@ -198,16 +200,30 @@ export default class EditionCard {
...
+          className="edition-card__card-info-link"
+          href={`${formatUrl(selectedLink.url, process.env.APP_ENV)}`}
+          onClick={() => gaUtils.trackGeneralEvent('Download', editionItem.source, work.title, '')}
+        >
+          Download
+        </a>
+      );
+    }
+
+    return undefined;
```
Comment (hokei) [inline src/app/components/Card/EditionCard.jsx:226]: this shouldn't be reachable

**Comment by hokei** [comment]
Verified that the events with labels show up on GA, but we probably need to deploy to have new triggers show up?  (My local work clicking doesn't seem to be adding new events yet)

**Comment by hokei** [comment]
:shipit:

File: src/app/components/Card/EditionCard.jsx (hunk 1 above)
Comment (mwbenowitz) [reply src/app/components/Card/EditionCard.jsx]: Just removed it in the merge to update the `formatUrl` calls

**Pull request merged**